modules/planning/conf/scenario/lane_follow_config.pb.txt

```diff
@@ -29,15 +27,6 @@ stage_config: {
     }
   }
 
```

Comment:
You previous commit to add these tasks haven't been pushed yet. 
Now you are deleting them, it's better to consolidate these changes since you are actually making no changes.

Reply:
Fixed. Thank you.